lib/index.js

```diff
@@ -42,6 +42,7 @@ var geocoderService;
  * @param {Function} [options.localGeocoder] A function accepting the query string which performs local geocoding to supplement results from the Mapbox Geocoding API. Expected to return an Array of GeoJSON Features in the [Carmen GeoJSON](https://github.com/mapbox/carmen/blob/master/carmen-geojson.md) format.
  * @param {'distance'|'score'} [options.reverseMode='distance'] - Set the factors that are used to sort nearby results.
  * @param {boolean} [options.reverseGeocode] Enable reverse geocoding. Defaults to false. Expects coordinates to be lat, lon.
+ * @param {Boolean|Object} [options.marker=true]  If `true`, a [Marker](https://docs.mapbox.com/mapbox-gl-js/api/#marker) will be added to the map at the location of the user-selected result using a default set of Marker properties.  If the value is an object, the marker will be constructed using these properties. If `false`, no marker will be added to the map.
```

Comment:
sorry to nit pick on you but https://docs.mapbox.com/mapbox-gl-js/api/#marker uses marker options not marker properties. Being consistent with terminology makes it easier for newcomers. So I prefer we use "options" rather than "properties" here.

Reply:
👍 Thanks for the thoughtful & thorough feedback! Consistency is good. Changing to `options`.